runtime/lua/vim/diagnostic.lua

```diff
@@ -794,8 +794,6 @@ end
 ---@param bufnr number Buffer number
 ---@param diagnostics table A list of diagnostic items |diagnostic-structure|. When omitted the
 ---                       current diagnostics in the given buffer are used.
----@param opts table Configuration table with the following keys:
----            - priority: Set the priority of the signs |sign-priority|.
```

Comment:
should we not mention `opts.signs` in the docstring here?

Reply:
Yes, we should. I meant to only remove the “priority” option, not the whole parameter